go.mod

```diff
@@ -45,9 +45,10 @@ require (
 	github.com/docker/go-units v0.5.0 // indirect
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.1.0-rc2 // indirect
-	golang.org/x/mod v0.9.0 // indirect
-	golang.org/x/tools v0.7.0 // indirect
+	golang.org/x/mod v0.10.0 // indirect
+	golang.org/x/tools v0.8.0 // indirect
 	gotest.tools/v3 v3.4.0 // indirect
+	mvdan.cc/gofumpt v0.5.0 // indirect
```

Comment:
Is there any reason why these dependencies were bumped? Can this be done in a separate PR